<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSelectionWindow</class>
 <widget class="QWidget" name="WidgetSelectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择控件</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="9,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="contentWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5">
      <item>
       <widget class="QListWidget" name="widgetList">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="rightWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="previewWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>200</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #ffffff;
border: 1px solid #cccccc;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="configGroupBox">
           <property name="title">
            <string> 控件配置</string>
           </property>
           <widget class="QWidget" name="formLayoutWidget">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>40</y>
              <width>391</width>
              <height>171</height>
             </rect>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <property name="horizontalSpacing">
              <number>0</number>
             </property>
             <property name="verticalSpacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="nameLabel">
               <property name="text">
                <string>控件名称:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="nameEdit"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="configPlaceholder">
               <property name="text">
                <string>更多配置:</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="buttonWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="confirmButton">
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
